nextflow/MaveDB/bin/map_scores_to_variants.py

```diff
@@ -80,7 +80,6 @@ def load_scores (f):
       # get index to sort data (required to match scores with MaveDB mappings)
       row['index'] = int(row['accession'].split("#")[1])
```

Comment:
Do we want to add the `index` here? it does not seem to have any use now.

Reply:
Removed, thanks!